templates/_layouts/_footer.html

```diff
@@ -0,0 +1,81 @@
+{% if is_docs %}
+<footer class="p-footer p-strip">
```

Comment:
I think it works better when footer is dark to diffrenciate from the content/sidebar above.

```suggestion
<footer class="p-footer p-strip is-dark">
```